preprocess_cloud_node:
  ros__parameters:
    range_based_filtering:
      active: True
      range: 500.0
    angle_based_filtering:
      active: False
      range: 10.0
    voxel_downsampling:
      active: True
      voxel_x: 0.05
      voxel_y: 0.05
      voxel_z: 0.05
plane_segmentation_node:
  ros__parameters:
    plane_segmentation:
      active: True
      distance_threshold: 0.1
    clip_ground:
      active: True
      height: 0.2
    vehicle_body_frame: "ego_vehicle"
    lidar_frame: "ego_vehicle/lidar"
clustering_node:
  ros__parameters:
    cluster_method: "euclidean"
    visualize_cluster: True
    euclidean_clustering:
      cluster_tolerance: 0.5
      min_cluster_size: 30
      max_cluster_size: 30000
